cmd/kops/delete.go

```diff
@@ -42,11 +42,11 @@ type DeleteOptions struct {
 
 var (
 	delete_long = templates.LongDesc(i18n.T(`
-	Delete Kubernetes clusters, instancegroups, and or secrets.
+	Delete Kubernetes clusters, instancegroups, and secrets, or a combination of the before mentioned.
```

Comment:
I think you mean "aforementioned".  I'd probably go with "or secrets", because nobody excepts XOR